go/packages/dev-mode/agent/forwarder.go

```diff
@@ -100,6 +110,48 @@ func FindRuntimeDone(logs []LogItem) *LogItem {
 	return nil
 }
 
+func tryParseBody(body *string) (string, string, int64) {
+	level := "INFO"
+
+	if body == nil {
+		return "", level, LogLevels[level]
+	}
+
+	// if this is AWS Node.js format, let's clean it
+	parts := strings.Split(*body, "\t")
+	if len(parts) > 3 {
+		b := strings.Join(parts[3:], "\t")
+		if _, ok := LogLevels[parts[2]]; ok {
+			level = parts[2]
+		}
+
+		return b, level, LogLevels[level]
+	}
+	// Check if we have a JSON object that contains a level or
+	// logLevel. This is copied from the frontend implementation
+	// we had left over from our OTEL days. This will reduce the
+	// amount of code we need on the frontend :)
+	var jsonMap map[string]interface{}
+	jsonParseError := json.Unmarshal([]byte(*body), &jsonMap)
+	if jsonParseError == nil {
+		logLevel1 := jsonMap["level"]
+		logLevel2 := jsonMap["logLevel"]
+		workingLevel := "INFO"
+		if logLevel1 != nil {
+			workingLevel = strings.ToUpper(logLevel1.(string))
+		} else if logLevel2 != nil {
+			workingLevel = strings.ToUpper(logLevel1.(string))
+		}
+		levelNumber, ok := LogLevels[workingLevel]
+		if ok {
+			return *body, workingLevel, levelNumber
+		}
+	}
+	//TODO: try other log formats
+
+	return *body, level, LogLevels[level]
+}
```

Comment:
I don't think I actually have to do this here let me double check 🤔

Reply:
We don't necessarily need this but I am going to leave it incase we ever decide we want to support publishing to a customers system or something. This will prevent the payloads leaving this service from being incorrect. It may be worth publishing this logic to a go package or something we so can share it between our backend/extension but that can be done later I guess 🤷